includes/Modules/Idea_Hub.php

```diff
@@ -181,6 +182,18 @@ public function register() {
 			 * Show admin notices on the posts page if we have saved / new ideas.
 			 */
 			add_filter( 'googlesitekit_admin_notices', $this->get_method_proxy( 'admin_notice_idea_hub_ideas' ) );
+
+			/**
+			 * Adds a special class name to idea posts.
+			 */
+			add_filter( 'post_class', $this->get_method_proxy( 'update_post_classes' ), 10, 3 );
+
+			add_action(
+				'admin_footer',
```

Comment:
Let's scope this to the specific screen rather than all admin screens.
```suggestion
				'admin_footer-edit.php',
```

Reply:
Updated.